public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -10,7 +10,7 @@ import { Branding } from 'app/core/components/Branding/Branding';
 import { DashboardGrid } from '../dashgrid/DashboardGrid';
 import { DashNav } from '../components/DashNav';
 import { DashboardSettings } from '../components/DashboardSettings';
-import { PanelEditor } from '../components/PanelEditor/PanelEditor';
+import { PanelEditor } from '../components/PanelEditor';
```

Comment:
think this should stay as `import { PanelEditor } from '../components/PanelEditor/PanelEditor';`

Reply:
also the UrlQueryValue import can now be removed from line 27